### DISCUSSION TITLE
how can i make the server up 24/7 in ubuntu

### DISCUSSION THREAD

**Discussion by amreahmed** [coder/code-server#6893] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS/Web Information

- Web Browser: chrome
- Local OS: ubuntu
- Remote OS: 
- Remote Architecture:
- `code-server --version`: last version

### Steps to Reproduce

1.
2.
3.

### Expected

i'm hosting code-server in my ubuntu but when i close the terminal it goes offilne is there is a way so i can make it up 24/7 like pm2 or something ?

### Actual

i'm hosting code-server in my ubuntu but when i close the terminal it goes offilne is there is a way so i can make it up 24/7 like pm2 or something ?

### Does this bug reproduce in native VS Code?

This cannot be tested in native VS Code

### Does this bug reproduce in GitHub Codespaces?

This cannot be tested in GitHub Codespaces

### Are you accessing code-server over a secure context?

- [ ] I am using a secure context.

**Comment by sanbei101** [comment]
`tmux`在窗口中运行`code-server`,当退出时按`Ctrl+B`退出终端就可以一直运行
